.gitmodules

```diff
@@ -10,6 +10,12 @@
 	shallow = true
 	active = false
 	ignore = all
+[submodule "scipy/sparse/linalg/_propack/PROPACK"]
```

Comment:
IIRC we intended to put it here because it is very specific to sparse SVD, and in the past libraries specific to a subpackage have been included in the corresponding folder. We put Boost in `_lib` because it is likely to be used in `stats` and `special`.

Reply:
ok fine with me to keep it here.

Reply:
Btw what does it do in `tools/docker_dev/gitpod.Dockerfile`? I'll add it if there is an example to follow.

Reply:
> Btw what does it do in `tools/docker_dev/gitpod.Dockerfile`? I'll add it if there is an example to follow.

You need to initialize the submodule following what's done L40-41, so I think here
`RUN git submodule update --init --depth=1 -- scipy/sparse/linalg/_propack/PROPACK`

Reply:
Why isn't this needed for UNU.RAN?

Reply:
Yeah I just merged this today as I saw it and we discussed how to fix it.